doc/user/content/sql/system-catalog.md

```diff
@@ -161,6 +161,19 @@ Field       | Type       | Meaning
 `name`      | [`text`]   | The name of the object.
 `type`      | [`text`]   | The type of the object: either `table`, `source`, `view`, `sink`, or `index`.
 
+### `mz_procs`
```

Comment:
Can we call this `mz_functions`? I've tried to keep the names of our catalog tables a bit more sensible than Postgres's.